.cirrus.yml

```
@@ -23,3 +22,39 @@ task:
     - . $HOME/.cargo/env
     - cargo test --target i686-unknown-freebsd
   before_cache_script: rm -rf $CARGO_HOME/registry/index
+
+task:
+  name: Linux
+  container:
+    - image: rust:1.24.1
```

I believe it shouldn't be a YAML list:

```suggestion
    image: rust:1.24.1
```

---

hehehe I beat you to it